tools/stub_generator/nvjpeg.json

```diff
@@ -0,0 +1,44 @@
+{
+   "extra_include":[
+      "<nvjpeg.h>"
+   ],
+   "return_type":"nvjpegStatus_t",
+   "calling_conv":"NVJPEGAPI",
+   "not_found_error":"NVJPEG_STATUS_IMPLEMENTATION_NOT_SUPPORTED",
+"functions": {
+      "nvjpegCreate": {},
+      "nvjpegDestroy": {},
```

Comment:
Do I understand correctly, that whenever we use some unlisted here nvjpeg function (or nvjpeg adds a function to the API), we should update this file?

Reply:
Yes we should, or if someone really wants can list all the functions nvJPEG has. I decided to do it lazily.